setup.cfg

```diff
@@ -79,3 +79,30 @@ per-file-ignores=
     # Docstrings are not required on tests.
     tests/*:D10,D20,D40
 
+[isort]
+skip=venv
+known_first_party=socketshark
+known_tests=tests
+sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,TESTS,LOCALFOLDER
+default_section=THIRDPARTY
+use_parentheses=true
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+line_length=79
+float_to_top=True
+
+[mypy]
+python_version = 3.8
```

Comment:
Just to confirm, do we still need to support 3.6? If so, we should typecheck with the oldest version.

Reply:
Indeed, set it to 3.6.